homeassistant/components/roborock/image.py

```diff
@@ -121,7 +127,16 @@ def _handle_coordinator_update(self):
     async def async_image(self) -> bytes | None:
         """Update the image if it is not cached."""
         if self.is_map_valid():
-            map_data: bytes = await self.cloud_api.get_map_v1()
+            response = await asyncio.gather(
+                *(self.cloud_api.get_map_v1(), self.coordinator.get_rooms()),
+                return_exceptions=True,
+            )
+            if not isinstance(response[0], bytes):
+                raise HomeAssistantError(
+                    translation_domain=DOMAIN,
+                    translation_key="map_failure",
+                )
+            map_data = response[0]
```

Comment:
So that actually isn't what is happening here. I am bundling together two requests - get_map_v1 and get_rooms() - I don't care about the response of get_rooms, I just want to call it if i can to update any room names - I could potentially remove this part anyways.

But I just need response[0] aka the response of get_map_v1